test/jdk/sun/security/ssl/EngineArgs/DebugReportsOneExtraByte.java

```diff
@@ -26,7 +26,7 @@
  * @bug 7126889
  * @summary Incorrect SSLEngine debug output
  * @library /test/lib /javax/net/ssl/templates
- * @run main DebugReportsOneExtraByte
+ * @run main/othervm DebugReportsOneExtraByte
```

Comment:
You don't have to run this in othervm mode. When there are no arguments, the test is relaunched in a new process (see lines 96-100).